src/nodes.coffee

```diff
@@ -979,6 +986,12 @@ exports.BooleanLiteral = class BooleanLiteral extends Literal
     value: if @value is 'true' then yes else no
     name: @originalValue
 
+exports.DefaultLiteral = class DefaultLiteral extends Literal
```

Comment:
Added a `DefaultLiteral` subclass so that it can have the proper AST (look like an `Identifier`)